administrator/components/com_privacy/Controller/DisplayController.php

```diff
@@ -74,10 +75,8 @@ public function display($cachable = false, $urlparams = [])
 				// For the default layout, we need to also push the action logs model into the view
 				if ($lName === 'default')
 				{
-					\JLoader::register('ActionlogsHelper', JPATH_ADMINISTRATOR . '/components/com_actionlogs/helpers/actionlogs.php');
-					BaseDatabaseModel::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_actionlogs/models', 'ActionlogsModel');
-
-					$logsModel = BaseDatabaseModel::getInstance('Actionlogs', 'ActionlogsModel');
+					$logsModel = Factory::getApplication()->bootComponent('Actionlogs')
```

Comment:
```suggestion
					$logsModel = $this->app->bootComponent('Actionlogs')
```
And remove these imports:
```
use Joomla\CMS\Factory;
use Joomla\CMS\MVC\Model\BaseDatabaseModel;
```